src/components/Search.jsx

```diff
@@ -0,0 +1,17 @@
+
+
+function Search() {
+
+  return (
+    <div className="Search">
+      <input className="input" type="text" placeholder="Search for any IP address or domain "/>
+      <button className="request">
```

Comment:
Maybe it is worth creating a separate Button component.

Reply:
what do you suggest?

Reply:
I did some work< but I'm not sure that I understand how does it work